ci/test/00_setup_env_i686_multiprocess.sh

```diff
@@ -13,6 +13,6 @@ export PACKAGES="llvm clang g++-multilib"
 export DEP_OPTS="DEBUG=1 MULTIPROCESS=1"
 export GOAL="install"
 export TEST_RUNNER_EXTRA="--v2transport"
-export BITCOIN_CONFIG="--enable-debug CC='clang -m32' CXX='clang++ -m32' \
-CPPFLAGS='-DBOOST_MULTI_INDEX_ENABLE_SAFE_MODE' CXXFLAGS='-Wno-error=documentation'"
+export BITCOIN_CONFIG="-DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_COMPILER='clang;-m32' -DCMAKE_CXX_COMPILER='clang++;-m32' \
+-DCMAKE_CXX_FLAGS='-DBOOST_MULTI_INDEX_ENABLE_SAFE_MODE -Wno-error=documentation'"
```

Comment:
Are these not going into APPEND flags because we don't add `-Wdocumentation` ourselves? Meaning that the primary use-case for APPEND_CXXFLAGS is specifically when we:
- Add a flag in our own buildsystem
- Want that flag turned off by the user at build-time

?

If so, it would be nice to document exactly that.

Reply:
Wait, we do add `-Wdocumentation` though. So why doesn't this have to be in APPEND_CXXFLAGS?

Reply:
We have to use `APPEND_CXXFLAGS` for `-Wno-documentation`. 

`-Wno-error=documentation` works with `CMAKE_CXX_FLAGS` just fine.

Reply:
Ah, ok. I would've thought that order matters there. But that makes sense.